aten/src/ATen/core/ATenDispatch.cpp

```diff
@@ -0,0 +1,10 @@
+#include <ATen/core/ATenDispatch.h>
+
+namespace at {
+
+ATenDispatch & globalATenDispatch() {
```

Comment:
Registering to the global table is not thread safe, right? Gotta be careful: library loads can happen in different threads.